app/models/room.rb

```diff
@@ -40,6 +42,16 @@ def number_with_type
     "#{number} (#{type})"
   end
 
+  # Sets the original suite to the current suite, unless it is already set
+  #
+  # @params [Hash] attrs Other attributes to update
+  #
+  # @return [Boolean] Whether or not the update was successful
+  def store_original_suite!(**attrs)
```

Comment:
What would happen if a suite is merged, unmerged, merged again, then unmerged again? It seems like the suite would be left with outdated info in the :original_suite attribute.